src/MTGCore.ClientApp/ClientApp/src/components/ResultsTable.vue

```diff
@@ -40,20 +35,7 @@ export default {
   },
   methods: {
     addToDeck(id) {
-      axios({
-        method: "post",
-        url: `https://localhost:44305/api/Deck/Add/1/${id}`,
-        data: this.$data
-      })
-        .then(() => {
-          this.someMethod();
-        })
-        .catch(err => {
-          alert(`There was an error submitting your form. See details: ${err}`);
-        });
-    },
-    someMethod(){
-        this.$parent.updateDeckList();
+      this.$emit("addCardToDeck", id);
```

Comment:
So is this calling the function from the parent App.vue?

Reply:
not really calling. Its creating an event with the name addCardToDeck with an ID value. Then the v:on command is used basicalled to say "whenever you see the addCardToDeck event run that named function"